src/index.tsx

```diff
@@ -512,3 +515,9 @@ export function createRoutesFromChildren(
 
   return routes;
 }
+
+type BreadCrumb = { breadcrumb?: string | ((param: any) => JSX.Element) | JSX.Element | null };
```

Comment:
Should the breadcrumb prop be any?
what other usecases might be missing.

Example what if people are using type:
( (param: any) => (param: any) => JSX.Element) ) 🤷‍♂️

Reply:
I think React.Node might work as a type? since it covers function components and strings I think.